src/index.html

```diff
@@ -6,9 +6,52 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="menu-link__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="menu-nav">
+        <ul class="menu">
+          <li class="menu-li">
+            <a class="menu-link is-active" href="#">APPLE</a>
```

Comment:
Let's add mock hrefs for all links
```suggestion
            <a class="menu-link is-active" href="#apple">APPLE</a>
```